### DISCUSSION TITLE
call action from another store

### DISCUSSION THREAD

**Discussion by josejachuf** [quasarframework/quasar#14074] [General - Components / Directives / etc] [answered]
Hi, I have two stores (foo, bar)

[quasar v1]

In foo actions I want to call a bar action. I understand what this should work, but it doesn't work

```
export async function myFooFn (context) {
  // my code
  context.dispatch('bar/myBarFn')
}
```

I get this error:

> unknown local action type: bar/myBarFn, global type: foo/bar/myBarFn

**Comment by MilosPaunovic** [comment, marked as answer]
You haven't shared important details like which state management library you are using, `Vuex` or `Pinia`. I'm guessing it's `Vuex`, and if that's the case, you need to call it with the `root` parameter set to `true`: `dispatch('anotherModule/anotherAction', {}, { root:true })`.

https://stackoverflow.com/questions/42984132/is-there-a-way-to-dispatch-actions-between-two-namespaced-vuex-modules

But, what it's more important, this is not Quasar related, so next time try to post it to relevant repository.

**Reply by josejachuf** [reply]
Works fine. Thanks
